website/docs/whats-new/release-notes/partials/figma-library-components.md

```diff
@@ -14,26 +14,9 @@
 
 ### May 7th, 2025
 
-This changelog corresponds with the v4.19 release.
```

Comment:
@jorytindall @LilithJames-HDS are these changes to the website release notes for figma expected?

Reply:
I will resolve this issue in a follow-up PR.